exec/csnex/Exec.cs

```diff
@@ -652,12 +657,22 @@ void INDEXAN()
 
         void INDEXDR()
         {
-            throw new NotImplementedException(string.Format("{0} not implemented.", MethodBase.GetCurrentMethod().Name));
+            ip++;
+            string index = stack.Pop().String;
+            Cell addr = stack.Pop().Address;
+            if (addr.Dictionary.ContainsKey(index) == false) {
```

Comment:
The `== false` here seems awkward. Use the `!` operator instead.

Reply:
Ok.  I wasn't sure when I wrote it that it sounded right; but since ContainsKey() returned a boolean, I went with the long-hand version.  I'll fix it.